src/LiveStorage.node.test.ts

```diff
@@ -0,0 +1,11 @@
+import { LiveStorage } from './LiveStorage'
+
+test('REAME example', () => {
+  const scenario = () => {
+    const posts = new LiveStorage('posts', [] as any[])
+    posts.update((prevPosts) => prevPosts.concat({ title: 'Brave new world' }))
+
+    expect(posts.value).toEqual([{ title: 'Brave new world' }])
+  }
+  expect(scenario).not.toThrow()
```

Comment:
Since we issue a real value-based assertion now, I think we can drop `not.toThrow()` one.

Reply:
aah, I removed it before the rebase and then forgot 🤦‍♂️ will fix

Reply:
Thank you!